src/components/GlobalHeader/RightContent.tsx

```diff
@@ -31,7 +41,7 @@ interface GeographicProps {
   city: ValueProps
 }
 
-interface CurrentUserProps {
+ interface CurrentUserProps {
```

Comment:
What's the change here? I think the whitespace. This shouldn't pass the linter. Please make sure your linter works when pushing the code.

Reply:
handled it. On update commit, no diff.
Will check with linter again